run.py

```diff
@@ -1,7 +1,14 @@
+import eventlet
+eventlet.monkey_patch()  # required for server to run properly
+
 from app import app, socketio
+from app.devices import measure_colors, sensor, led
+
+
+if __name__ == '__main__':
+    # run the sensor and led control in a separate coroutine
+    eventlet.spawn(measure_colors, socketio, sensor, led)
 
-# This is required to run app as a socketio server
-# Without this websocket cannot be used
-socketio.run(app, host='0.0.0.0', port=8080)
-# app.run(host='0.0.0.0', port=8080)
+    # Without this websocket cannot be used
+    socketio.run(app, host='0.0.0.0', port=8080)
```

Comment:
Czy na pewno zostawiamy serwer na `8080`? Czy to pozostałość po developmencie na labach?